src/OrchardCore.Themes/TheBlogTheme/Views/Content-BlogPost.Summary.liquid

```diff
@@ -1,18 +1,18 @@
 ﻿<div class="post-preview">
-    {% a display_for:ContentItem %}
+    {% a display_for:Model.ContentItem %}
         <h2 class="post-title">
-            {{ ContentItem.Content.TitlePart.Title }}
+            {{ Model.ContentItem.Content.TitlePart.Title }}
         </h2>
-        {% if ContentItem.Content.BlogPost.Subtitle.Text %}
+        {% if Model.ContentItem.Content.BlogPost.Subtitle.Text %}
```

Comment:
No, if Text is empty then it will still render the tags. By change, if you didn't set anything it will be null, but try to save the content, then delete the text.